keyboards/handwired/scottokeebs/scottoergo/info.json

```diff
@@ -0,0 +1,87 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoErgo",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "GP0",
+            "GP1",
+            "GP2",
+            "GP3",
+            "GP4",
+            "GP22",
+            "GP21",
+            "GP20",
+            "GP19",
+            "GP18"
+        ],
+
+        "rows": ["GP12", "GP13", "GP14", "GP15"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x10_6": {
```

Comment:
Entirely optional, but changing `LAYOUT_ortho_3x10_6` to `LAYOUT_split_3x5_3` will allow community layouts to automatically be assigned (namely miryoku).  
Would have to apply this change to all `keymap.c` files also.

Reply:
Changed in [d67b78c](https://github.com/qmk/qmk_firmware/pull/20944/commits/d67b78c9425552a43f8df44d87a577469d26a0e0).